prow/cmd/horologium/main.go

```diff
@@ -100,20 +101,26 @@ func sync(kc kubeClient, cfg *config.Config, cr cronClient, now time.Time) error
 
 	errs := []error{}
 	for _, p := range cfg.Periodics {
+		max := 1
+		if p.MaxConcurrency != nil {
+			max = *p.MaxConcurrency
+		}
 		j, ok := latestJobs[p.Name]
-
-		if p.Cron == "" {
-			if !ok || (j.Complete() && now.Sub(j.Status.StartTime.Time) > p.GetInterval()) {
-				if _, err := kc.CreateProwJob(pjutil.NewProwJob(pjutil.PeriodicSpec(p), p.Labels)); err != nil {
-					errs = append(errs, err)
+		runningJobs := currentJobs[p.Name]
+		if max == 0 || len(runningJobs) < max {
```

Comment:
Let's used a typed constant for `0` here in the api package to clarify it means unbounded